deploy/catalog/templates/catalog.yaml

```diff
@@ -15,6 +15,24 @@ spec:
         imagePullPolicy: Always
         ports:
         - containerPort: 10000
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10000
+          failureThreshold: 3
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 2
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10000
+          failureThreshold: 1
```

Comment:
just curious whether just one failure to declare not ready is optimal (I don't have enough experience with k8s probes to tell). It is great starting point (imo) and we can tweak as needed.